src/main/java/se/bjurr/violations/lib/example/CopyOfMyClass.java

```diff
@@ -0,0 +1,19 @@
+package se.bjurr.violations.lib.example;
```

Comment:
Severity: INFO
File: /home/bjerre/workspace/violations-test/src/main/java/se/bjurr/violations/lib/example/CopyOfMyClass.java
Reporter: CHECKSTYLE
Must have at least one statement.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>